ext/zend_test/test.stub.php

```diff
@@ -16,6 +18,7 @@ function zend_test_nullable_array_return(): ?array {}
 
 function zend_test_void_return(): void {}
 
+/** @deprecated since 5.0 */
```

Comment:
It turned out that `@deprecated` alone isn't enough, that's why I added a random number here.

Reply:
Isn't enough for what?

Reply:
Sorry for being obscure: when I didn't specify a value for the tag, `ZEND_FE` was generated instead of `ZEND_DEP_FE`. My guess was that it's related to php-parser, but I didn't bother to check it.

Reply:
You probably need to add a `?` at the end of the regex in https://github.com/php/php-src/blob/96c9743898e3db445cae205555d6443ce6e3cb55/build/gen_stub.php#L425 to make the argument optional.

Reply:
Thanks! Then my guess was not right :)